src/memory_tracker-inl.h

```diff
@@ -255,6 +256,12 @@ void MemoryTracker::TrackField(const char* name,
   TrackField(name, value.GetJSArray(), "AliasedBuffer");
 }
 
+void MemoryTracker::TrackField(const char* name,
+                               const sockaddr* addr,
+                               const char* node_name) {
+  TrackFieldWithSize(name, SocketAddress::GetLength(addr), node_name);
```

Comment:
This is really to enable tracking of the various std::unordered_maps used in QuicSocket. Because we use sockaddr* as a key, TrackField fails if there's not an implementation for this. There may be something I'm missing about the memory tracker implementation tho.